verification/CMakeLists.txt

```diff
@@ -214,16 +216,11 @@ create_dsdl_target(dsdl-test
 
 add_dependencies(dsdl-test nunavut-support)
 
-# +---------------------------------------------------------------------------+
-# | FLAG SETS
-# +---------------------------------------------------------------------------+
-string(TOLOWER ${NUNAVUT_VERIFICATION_LANG} LOCAL_VERIFICATION_LANG)
-
-if (LOCAL_VERIFICATION_LANG STREQUAL "c")
-     set(LOCAL_VERIFICATION_LANG_STANDARD_C "${NUNAVUT_VERIFICATION_LANG_STANDARD}")
-     set(LOCAL_VERIFICATION_LANG_STANDARD_CXX "")
-     list(APPEND LOCAL_ADDITIONAL_DSDL_LIBS "")
-elseif(LOCAL_VERIFICATION_LANG STREQUAL "cpp")
+if(LOCAL_VERIFICATION_LANG STREQUAL "cpp")
```

Comment:
Just refactoring in this file. No logic change. I wanted to separate out the section that determined compiler flags from the section that added in c-type support for the c++ tests.